### PR TITLE
refactor(elements|ino-icon): rename icon `request` to `share`

### LINKED ISSUE #377: Add new icon "share"
For a new sharing feature within our internal product development, we need a "share" icon to provide the user an appropriate experience.

**Acceptance criteria**
* Rename icon

### PR DESCRIPTION
**Pull request by BenPag**
Closes #377

## Proposed Changes

- rename icon `request` to `share`

## Things to check

- [ ] Does the change need to be documented?
- [ ] Does any existing example code needs to be updated?
- [ ] Is the change properly tested?
- [ ] Is it helpful to provide another example to demonstrate the new feature?
- [ ] Are there other code lines that need to be modified?

### CODE REVIEW

File: packages/elements/src/components/ino-icon/icons.js [hunk 1]
```diff
@@ -1,9 +1,5 @@
 export default [
-  '_fab_set_arrow_down',
```
Comment (janivo) [inline packages/elements/src/components/ino-icon/icons.js:2]: Why did the order change? Did you use the `integrate:icons` script or did you change it manually?

Comment (BenPag) [reply packages/elements/src/components/ino-icon/icons.js:2]: I used the `integrate:icons` script. Maybe this "issue" depens on my operating system (Windows)?

Comment (janivo) [reply packages/elements/src/components/ino-icon/icons.js:2]: Could be, but doesn't really matter

**Comment by janivo** [comment]
Could you resolve the conflicts @BenPag ?

**Pull request merged**